### ISSUE TITLE
[BUG] Flags are not being passed to serve command

### ISSUE THREAD

**Issue by inabhi9** [Permify/permify#1182]
**Describe the bug**
Flags are not being passed to `serve` command. I observe this behaviour in version 0.7.7 -> 0.7.9

**To Reproduce**

```
❯ ./permify-v0.7.9 serve --http-enabled=false --grpc-port=12345 --account-id=1
┌────────────────────────────────────────────────────────┐
│                    Permify v0.7.9                      │
│          Fine-grained Authorization Service            │
│                                                        │
│    docs: ............... https://docs.permify.co       │
│    github: .. https://github.com/Permify/permify       │
│    blog: ............... https://permify.co/blog       │
│                                                        │
└────────────────────────────────────────────────────────┘
time=2024-03-28T19:06:59.560+01:00 level=INFO msg="🚀 starting permify service..."
time=2024-03-28T19:06:59.561+01:00 level=ERROR msg="Account ID is not set. Please fill in the Account ID for better support. Get your Account ID from https://permify.co/account"
time=2024-03-28T19:06:59.563+01:00 level=INFO msg="🚀 grpc server successfully started: 3478"
time=2024-03-28T19:06:59.563+01:00 level=INFO msg="🚀 invoker grpc server successfully started: 5000"
time=2024-03-28T19:06:59.570+01:00 level=INFO msg="🚀 http server successfully started: 3476"
```
correct behaviour with v0.7.6

```
❯ ./permify-v0.7.6 serve --http-enabled=false --grpc-port=12345 --account-id=1
┌────────────────────────────────────────────────────────┐
│                    Permify v0.7.6                      │
│          Fine-grained Authorization Service            │
│                                                        │
│    docs: ............... https://docs.permify.co       │
│    github: .. https://github.com/Permify/permify       │
│    blog: ............... https://permify.co/blog       │
│                                                        │
└────────────────────────────────────────────────────────┘
time=2024-03-28T19:05:10.678+01:00 level=INFO msg="🚀 starting permify service..."
time=2024-03-28T19:05:10.680+01:00 level=INFO msg="🚀 grpc server successfully started: 12345"
time=2024-03-28T19:05:10.680+01:00 level=INFO msg="🚀 invoker grpc server successfully started: 5000"
time=2024-03-28T19:05:12.267+01:00 level=INFO msg="gracefully shutting down"
```

**Expected behavior**
It should respect the flags.

**Additional context**
`config` command shows correct flags

**Environment (please complete the following information, because it helps us investigate better):**
 - OS: macOS
 - Version 0.7.7 -> 0.7.9

**Comment by tolgaOzen** [comment]
Hello @inabhi9, we fixed this issue in version v0.8.0. Can you test it?

**Comment by inabhi9** [comment]
It's working as expected in v0.8.0!

Cheers 🎉
